README.md

```diff
@@ -30,16 +37,22 @@ $ dbt seed
 
 ### Running Instructions:
 
-Run fal flow
+1. Run `dbt run`
 
 ```bash
-$ fal flow run --experimental-models
-## runs the whole graph
+$ dbt run
+## Runs the whole graph including the Python models
```

Comment:
Does not run the notification, is that part of the "whole graph"?

Reply:
Not really, I think its good separate transformations vs non-transformations.

Reply:
```suggestion
## Runs the SQL models on the datawarehouse and Python models locally with fal
```